File: src/ds-visuals/snippets/min-heap/delete.java
int delete(){
	int val = this.heap[1];
	if(this.size== 0){
		System.out.println("Heap Empty");
		return -1;
	}
	this.heap[1] = this.heap[this.size];
	this.size--;
	this.adjust();
	return val;
}

File: src/ds-visuals/snippets/max-heap/delete.c
8
very_fast =
fast =  0, 1, 2, 8
medium = 3, 4, 5, 6, 7
slow = 
very_slow = 

int delete(Heap* heap){
	int val = heap->arr[1];
	if(heap->size == 0){
		printf("Heap Empty\n");
		return -1;
	}
	heap->arr[1] = heap->arr[heap->size];
	heap->size--;
	adjust(heap);
	return val;
}